core/src/main/java/io/questdb/griffin/model/CreateTableModel.java

```diff
@@ -174,6 +174,11 @@ public CharSequence getTableName() {
         return name.token;
     }
 
+    @Override
+    public CharSequence getTargetTableName() {
```

Comment:
can we call it `tableName` ? Target table name is a new term

Reply:
Switched to getTableName() even though that forced use of Object return type because method exists in classes derived from ExecutionModel, but with different return types .

Reply:
This is not my ideal way to solve the problem. I think `instanceOf` should be avoided at all costs as it could have massive scalability implications. I tried to play around to see how can we simplify this `tableName` knot and got carried away a but. So here is commit that I think solves the problem nicely :) https://github.com/questdb/questdb/pull/2680/commits/3e3353850b1e4ac5e2beb402e37e778b408e14d1